src/NuGet.Core/NuGet.Protocol.Core.v3/HttpSource/HttpSource.cs

```diff
@@ -45,6 +44,8 @@ public class HttpSource : IDisposable
 
         public TimeSpan DownloadTimeout { get; set; } = DefaultDownloadTimeout;
 
+        public IHttpRetryHandler RetryHandler { get; set; } = new HttpRetryHandler();
```

Comment:
Since this is for test use only, expand the properties in full, and add XML docs, telling the user that set is for unit tests only